### DISCUSSION TITLE
gongju/slf4j

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#176] [Announcements]
# gongju/slf4j

我在读嵩山版的阿里巴巴开发手册（没有的小伙伴，记着找我要）的时候，就发现了一条「强制」性质的日志规约： 应用中不可以直接使用日志系统（Log4j、Logback）中的 API，而应该使用日志框架中的 API，比如说 SLF4J，使用门面模式的日志框架，有利于维护和统一各个类的日志处理方式。 （为什么我把这段文字手敲了下来呢，因为我发现阿里巴巴开发手册上的有语病，瞧下面红色标出的部分）

https://tobebetterjavaer.com/gongju/slf4j.html

**Comment by qinple** [comment]
666

**Comment by isjiamu** [comment]
666
